src/app/reactUtils.ts

```diff
@@ -245,4 +245,10 @@ export const disableContentTabbingHandler = () => {
   };
 };
 
-export const useDisableContentTabbing = () => React.useEffect(disableContentTabbingHandler);
+export const useDisableContentTabbing = (disableTabbing: boolean) => {
+  React.useEffect(() => {
+    if (disableTabbing) {
+      disableContentTabbingHandler();
+    }
```

Comment:
`disableContentTabbingHandler()` is returning a function that should be called in the `useEffect` for cleaning up effect
sth like 
```js
const cleanUp = disableContentTabbingHandler();
return cleanUp;
```
but since this is in `if` then I think it will mess up with how hooks are working
I think it would be better to do this the same way as with `useFocusLost`